BasicGates/Tasks.qs

```diff
@@ -111,24 +111,34 @@ namespace Quantum.Kata.BasicGates {
         // ...
     }
 
+    // Task 1.7. Global Phase Change
+    // Inputs:
+    //     1) Pauli operator (μ) represented as Pauli.
+    //     2) Angle alpha, in radians, represented as Double.
+    //     3) A qubit in state β|0⟩ + γ|1⟩.
+    // Goal: Change the state of the qubit to - β|0⟩ - γ|1⟩.
+    operation GlobalPhaseChange(pauli: Pauli, alpha: Double, q: Qubit) : Unit is Adj+Ctl{
```

Comment:
If the task is just to add a `-1` global phase to the qubit state, you don't need to pass the first two of these parameters - just the qubit itself. 
```suggestion
    operation GlobalPhaseChange (q: Qubit) : Unit is Adj + Ctl {
```
This way the reference solution stays the same, `R(PauliI, 2.0 * PI(), q);`, but there are fewer parameters to deal with (the way it is phrased now, the purpose of these parameters is unclear, and you only test the solution with one value). 

If you're worried that people will not be able to find `PI()` function, you can add a hint telling them the namespace they need to open for it (they'll need to use the constant with fairly high precision - I've seen solutions that take 3.1415 fail being not accurate enough).

In addition, this change allows people to write other solutions - for example, ZXZX will add a -1 phase without any rotations and constants required.